parser.py

```diff
@@ -110,7 +112,7 @@ def lookup_users(user_ids, users):
         with requests.Session() as session:
```

Comment:
This line is now misleading because many of these are in state and will get filtered out in get_twitter_users():

print(f'{len(filtered_user_ids)} users are unknown.')

Reply:
As we're removing the duplication of `users = {}`, sure. We can do the filtering earlier... Also saves us from passing the dict pointer three layers deep...